src/Integration.UnitTests/app.config

```diff
@@ -78,6 +78,10 @@
         <assemblyIdentity name="System.Interactive.Async" publicKeyToken="94bc3704cddfc263" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-3.0.3000.0" newVersion="3.0.3000.0" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="SonarAnalyzer" publicKeyToken="c5b62af9de6d7244" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-6.7.0.0" newVersion="6.7.0.0" />
+      </dependentAssembly>
```

Comment:
Any idea why are these binding redirects required now when they weren't required before?

Reply:
No idea. VS added it.